src/main/com/java/textFields/TextFields.java

```diff
@@ -1,4 +1,4 @@
-package main.com.java.textFields;
```

Comment:
Name of the package should be textFields and not textfields, because of data normalization between branches.